### PR TITLE
testing/xsel: fixed source + build

### PR DESCRIPTION
**Pull request by jjsearle**

### CODE REVIEW

File: testing/xsel/APKBUILD [hunk 1]
```diff
@@ -2,18 +2,17 @@
...
-makedepends="libxt-dev"
+makedepends="libxt-dev autoconf automake libtool"
 subpackages="$pkgname-doc"
-source="http://www.vergenet.net/~conrad/software/$pkgname/download/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir/$pkgname-$pkgver"
+source="$pkgname-$pkgver.tar.gz::https://github.com/kfish/xsel/archive/$pkgver.tar.gz"
 
 build() {
-	cd "$builddir"
+	./autogen.sh
```
Comment (maxice8) [inline testing/xsel/APKBUILD]: please separate the ./autogen.sh into prepare()

Comment (jjsearle) [reply testing/xsel/APKBUILD]: autogen.sh separated into prepare function.

File: testing/xsel/APKBUILD [hunk 2]
```diff
@@ -2,18 +2,24 @@
...
-source="http://www.vergenet.net/~conrad/software/$pkgname/download/$pkgname-$pkgver.tar.gz"
-builddir="$srcdir/$pkgname-$pkgver"
+source="$pkgname-$pkgver.tar.gz::https://github.com/kfish/xsel/archive/$pkgver.tar.gz"
+
+prepare() {
+	aclocal
+	libtoolize --automake
+	autoheader
+	automake --add-missing
+	autoconf
```
Comment (Cogitri) [inline testing/xsel/APKBUILD]: `autoreconf -if` doesn't cut it?

Comment (nmeum) [reply testing/xsel/APKBUILD]: @jjsearle :question:

Comment (maxice8) [reply testing/xsel/APKBUILD]: can't it all be replaced with a simple call to autoreconf -fi like it is done elsewhere ?

**Pull request closed without merging**